### PR TITLE
fix(web): user list when sharing an album

### PR DESCRIPTION
**Pull request by martabal**
| Before | After |
| :----: | :---: |
|<video src="https://github.com/immich-app/immich/assets/74269598/9b702948-6ef2-4b4f-8bcd-76034c18d85a"> | <video src="https://github.com/immich-app/immich/assets/74269598/e70c64da-aaf8-4cc5-913c-bfee94c87ded">  |

### CODE REVIEW

File: open-api/immich-openapi-specs.json [hunk 1]
```diff
@@ -6221,7 +6221,7 @@
   "info": {
     "title": "Immich",
     "description": "Immich API",
-    "version": "1.93.0",
+    "version": "1.93.1",
```
Comment (danieldietzler) [inline open-api/immich-openapi-specs.json:6224]: I don't think this PR should update this

File: web/src/lib/components/album-page/user-selection-modal.svelte [hunk 2]
```diff
@@ -92,10 +92,11 @@
             class="flex w-full place-items-center gap-4 px-5 py-4 transition-all hover:bg-gray-200 dark:hover:bg-gray-700"
           >
             {#if selectedUsers.includes(user)}
-              <span
-                class="flex h-12 w-12 place-content-center place-items-center rounded-full border bg-immich-primary text-3xl text-white dark:border-immich-dark-gray dark:bg-immich-dark-primary dark:text-immich-dark-bg"
-                >✓</span
+              <div
+                class="flex h-10 w-10 items-center justify-center rounded-full border bg-immich-primary text-3xl text-white dark:border-immich-dark-gray dark:bg-immich-dark-primary dark:text-immich-dark-bg"
               >
+                <p>✓</p>
```
Comment (danieldietzler) [inline web/src/lib/components/album-page/user-selection-modal.svelte:98]: Why is there just an empty p tag?

Comment (martabal) [reply web/src/lib/components/album-page/user-selection-modal.svelte:98]: It's not empty (✓)

File: open-api/immich-openapi-specs.json (hunk 1 above)
Comment (martabal) [reply open-api/immich-openapi-specs.json]: It got updated automatically with `git checkout main`

**Review by alextran1502 — APPROVED** [review]

**Comment by alextran1502** [comment]
Can you help rebase the PR?

**Comment by martabal** [comment]
> Can you help rebase the PR?

Sorry, you're too fast

**Pull request merged**